docs/source/administrator/upgrading/index.md

```diff
@@ -1,38 +1,40 @@
-# Upgrading your Helm chart
+# Upgrading JupyterHub for Kubernetes
 
-This page covers best-practices in upgrading your JupyterHub deployment via updates
+This section covers best-practices in upgrading your JupyterHub deployment via updates
 to the Helm Chart.
 
-```{toctree}
-:maxdepth: 1
-:caption: Upgrade Guide
-
-upgrade-1-to-2
-```
+Z2JH follows [semantic versioning](https://semver.org/), with each version taking the form `MAJOR.MINOR.PATCH`.
+Minor and patch releases should be backwards compatible, and shouldn't require changes to your deployment.
+Review the [CHANGELOG](changelog) to find out about new features or bug fixes that affect your deployment,
+then follow [](helm-upgrade-command).
```

Comment:
I'm not sure how the `[](helm-upgrade-command)` part will render, but it seems like a forgotten title for the link - or is sphinx doing some magic here?

Reply:
It's magic, it uses the subheading for that label.

Reply:
Ah nice!